drivers/input/input_cst816s.c

```diff
@@ -88,6 +93,18 @@ struct cst816s_config {
 #ifdef CONFIG_INPUT_CST816S_INTERRUPT
 	const struct gpio_dt_spec int_gpio;
 #endif
+
+#ifdef CONFIG_PM_DEVICE
+	/** cst816s power management profile. */
```

Comment:
Nit: it would be good to document also the struct fields.

Reply:
Will do